SwiftUI_Homework/SwiftUI_Homework/TabView/PrivacyView.swift

```diff
@@ -0,0 +1,193 @@
+//
+//  Privacy.swift
+//  SwiftUI_Homework
+//
+//  Created by Анастасия Козлова on 22.01.2023.
+//
+
+import SwiftUI
+
+/// Экран выбора тарифа
+struct PrivacyView: View {
+    
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let blueColor = Color(#colorLiteral(red: 0.2569369539, green: 0.8115655123, blue: 0.9686274529, alpha: 0.4798957875))
+        static let privacyMattersText = "Privacy Matters"
+        static let protectText = "Protect your online activate with VPN Plus"
+        static let dayText = "7 days"
+        static let dayMoneyText = "$3"
+        static let monthText = "1 month"
+        static let monthMoneyText = "$10"
+        static let infinityText = "Infinity"
+        static let infinityMoneyText = "$150"
+        static let buyText = "Buy"
+        static let sevenImageName = "seven"
+        static let monthImageName = "month"
+        static let allImageName = "all"
+        static let spacingValue: CGFloat = 20
+        static let widthImageValue: CGFloat = 150
+        static let heightImageValue: CGFloat = 150
+        static let widthButtonValue: CGFloat = 240
+        static let heightButtonValue: CGFloat = 50
+        static let cornerRadiusValue: CGFloat = 15
+        static let paddingValue: CGFloat = 10
+        static let paddingButtonValue: CGFloat = 20
+    }
+    
+    // MARK: - Body
+    
+    var body: some View {
+        ZStack {
+            Color.black.ignoresSafeArea(edges: .top)
+            ScrollView {
+                VStack(spacing: Constants.spacingValue) {
+                    mainTextView
+                    descriptionTextView
+                    sevenImageView
+                    sevenButtonView
+                    monthImageView
+                    monthButtonView
+                    infinityImageView
+                    infinityButtonView
```

Comment:
```suggestion
                    sevenImageView
                    sevenButtonView
                    monthImageView
                    monthButtonView
                    infinityImageView
                    infinityButtonView
```
здесь получается много одинакового кода, можно было сделать функцию или вынести в отдельную вью с vstack image+button и получится одна вью вместо 6